articles/aks/auto-upgrade-cluster.md

```diff
@@ -22,6 +22,14 @@ Cluster auto-upgrade provides a set once and forget mechanism that yields tangib
 
 AKS follows a strict versioning window with regard to supportability. With properly selected auto-upgrade channels, you can avoid clusters falling into an unsupported version. For more on the AKS support window, see [Alias minor versions][supported-kubernetes-versions].
 
+## Customer versus AKS-initiated auto-upgrades
+
+Customers can specify cluster auto-upgrade specifics in the guidance below. These upgrades will occur based on the cadence that the customer specifies and are recommended for customers to remain on supported kubernetes versions.
+
+AKS will also initiate auto-upgrades for unsupported clusters. All clusters with kubernetes versions N-3 and below will be automatically upgraded to remain in an AKS support [policy][supported-kubernetes-versions].
```

Comment:
We probably need to clarify this further. When a cluster in a n-3 version (where n is the latest supported AKS GA version) is about to drop to n-4 AKS will automatically upgrade the cluster to n-2

Reply:
As you kind of have after but folks will panic on the first one likely.